app/views/user_mailer/analytics.text.haml

```diff
@@ -1,16 +1,19 @@
-= "Your weekly summary for #{@stats[:since].strftime('%m/%d')} - #{@stats[:till].strftime('%m/%d')}"
+= raw t(:"email.analytics.title", since: @stats[:since].strftime('%d %B'))
 = '---------------------------------------------------------------------------'
 
 = raw t(:"email.analytics.stats", @stats)
 
-= "Name | Motions created | Discussions created | Comments created | Votes created"
-= '---------------------------------------------------------------------------'
+- if @stats[:has_activity]
+  = "Name | Motions created | Discussions created | Comments created | Votes created"
```

Comment:
I guess this should be translated?

Also you could use tab characters to align the columns, maybe.